### PR TITLE
added .section to css and HTML

### PR DESCRIPTION
**Pull request by ChadESmith42**
@eschwartz I formatted the HTML to make it consistent and remove some of the extra whitespace. I also put your content into its own `<section>` tag. I added one CSS class to the HTML.

Please review and approve at your convenience.

### CODE REVIEW

**Comment by eschwartz** [comment]
Hey @ChadESmith42, checking this out with the whole class, here they are:
Screenshot_2021-07-01-13 26 04

**Comment by ChadESmith42** [comment]
@eschwartz That is the most suspect group of miscreants and sneaky characters ....

File: public/index.html [hunk 1]
```diff
@@ -1,43 +1,34 @@
...
-
-
+    <link rel="stylesheet" href="./styles/style.css" />
+  </head>
+  <body>
+    <main>
+      <section id="aboutChad" class="about section">
+        <h2>About Chad</h2>
+        <ul>
+          <li><span>Likes</span>Loves ice cream.</li>
```
Comment (eschwartz) [inline public/index.html:15]: What if.... we added a `strikethrough` class to this `<span>` so we know why it's here?

File: public/index.html [hunk 2]
```diff
@@ -1,43 +1,34 @@
...
-        </li>
-      </ul>
-
-
-
+    <link rel="stylesheet" href="./styles/style.css" />
+  </head>
+  <body>
+    <main>
+      <section id="aboutChad" class="about section">
```
Comment (eschwartz) [inline public/index.html:12]: why a section tagname, and a classname? isn't that redundant?

File: public/index.html (hunk 1 above)
Comment (eschwartz) [reply public/index.html:15]: For example:
```html
<span class="strikethrough">Likes</span>
```

File: public/index.html (hunk 2 above)
Comment (eschwartz) [reply public/index.html:12]: I know it's silly, but the airbnb standards say not to use a class name that matches the tag name. 🤷

File: public/index.html (hunk 1 above)
Comment (ChadESmith42) [reply public/index.html]: I added a "strike-through" class and applied it to the `<span>`.

**Review by eschwartz — APPROVED** [review]
image

**Comment by eschwartz** [comment]
That was a journey, nice work
image

**Comment by ChadESmith42** [comment]
May have stopped believing at some point.

**Pull request merged**